modules/@apostrophecms/module/lib/events.js

```diff
@@ -4,6 +4,15 @@ module.exports = function(self) {
 
   return {
 
+    // Register an old event name to be accepted for a new event name.
+    // If newName is emitted, oldName will be accepted as a synonym in handlers.
+    // Currently supported for Apostrophe core module events only. Not to be
+    // used for module-specific events. Call with just the event name,
+    // not the `apostrophe:` part.
+    alias(oldName, newName) {
```

Comment:
Let's call it `aliasEvent` since when it's called there's no connection to this file.

Reply:
Done.